packages/react-tooltip-trigger/src/components/TooltipTrigger/renderTooltipTrigger.tsx

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
+import * as ReactDOM from 'react-dom';
+import { getSlots } from '@fluentui/react-utilities';
+import { TooltipTriggerState } from './TooltipTrigger.types';
+import { tooltipTriggerShorthandProps } from './useTooltipTrigger';
+
+/**
+ * Render the final JSX of TooltipTrigger
+ * {@docCategory TooltipTrigger}
+ */
+export const renderTooltipTrigger = (state: TooltipTriggerState) => {
+  const { slots, slotProps } = getSlots(state, tooltipTriggerShorthandProps);
+
+  return (
+    <>
+      {state.children}
+      {ReactDOM.createPortal(<slots.tooltip {...slotProps.tooltip} />, state.portalRoot)}
```

Comment:
```suggestion
      {isSSR() ? null : ReactDOM.createPortal(<slots.tooltip {...slotProps.tooltip} />, state.portalRoot)}
```

We haven't focus on SSR rendering yet, but IMO we should be ready for it